bibliopixel/drivers/serial/driver.py

```diff
@@ -92,7 +92,7 @@ def _connect(self):
                     pass
                 else:
                     self._bufPad = BufferChipsets[
-                        self._ledtype](self.numLEDs) * 3
+                        self._ledtype](self.numLEDs) * self.pixel_size
```

Comment:
This is problematic because the AllPixel is absolutely only 3 bytes per pixel. The protocol I created doesn't support anything else. For now I'd rather just throw an error if you try to use something else with Serial. That driver will need massive rework to do the right thing for each device. So change this to an exception if pixel_size != 3 and I'm happy for now.